test/e2e/specs/site-editor/pages.spec.js

```diff
@@ -18,244 +18,244 @@ async function draftNewPage( page ) {
-		await expect(
-			page.locator(
-				'role=button[name="Dismiss this notice"i] >> text="Edit your template to edit this block."'
-			)
-		).toBeVisible();
+async function testToggleTemplatePreview( editor, page ) {
+	await expect(
+		editor.canvas.getByRole( 'document', {
+			name: 'Block: header',
+		} )
+	).toBeVisible();
+	await expect(
+		editor.canvas.getByRole( 'document', {
+			name: 'Block: Content',
+		} )
+	).toBeVisible();
 
-		// Switch to template editing focus.
-		await editor.openDocumentSettingsSidebar();
-		await page
-			.getByRole( 'region', { name: 'Editor settings' } )
-			.getByRole( 'button', { name: 'Template options' } )
-			.click();
-		await page.getByRole( 'button', { name: 'Edit template' } ).click();
-		await expect(
-			editor.canvas.getByRole( 'document', {
-				name: 'Block: Content',
-			} )
-		).toContainText(
-			'This is the Content block, it will display all the blocks in any single post or page.'
-		);
-		await expect(
-			page.locator(
-				'role=button[name="Dismiss this notice"i] >> text="You are editing a template."'
-			)
-		).toBeVisible();
+	// Ensure order is preserved between toggling.
+	await page
+		.locator(
+			'[aria-label="Block: Content"] + [aria-label="Block: Title"]'
+		)
+		.isVisible();
 
-		// Edit a block that's in the template.
-		await editor.canvas
-			.getByRole( 'textbox', { name: 'Site title text' } )
-			.fill( 'New Site Title' );
+	await editor.openDocumentSettingsSidebar();
+	// Toggle template preview to "off".
+	const templateOptionsButton = page
+		.getByRole( 'region', { name: 'Editor settings' } )
+		.getByRole( 'button', { name: 'Template options' } );
+	await templateOptionsButton.click();
+	const templatePreviewButton = page
+		.getByRole( 'menu', { name: 'Template options' } )
+		.getByRole( 'menuitem', { name: 'Template preview' } );
 
-		// Go back to page editing focus.
-		await page
-			.getByRole( 'region', { name: 'Editor top bar' } )
-			.getByRole( 'button', { name: 'Back' } )
-			.click();
+	await expect( templatePreviewButton ).toHaveAttribute(
+		'aria-pressed',
+		'true'
+	);
+	await templatePreviewButton.click();
+	await expect( templatePreviewButton ).toHaveAttribute(
+		'aria-pressed',
+		'false'
+	);
 
-		// Site Title and Page entities should have been modified.
-		await page.getByRole( 'button', { name: 'Save', exact: true } ).click();
-		await expect(
-			page.locator(
-				'role=region[name="Save panel"] >> role=checkbox[name="Title"]'
-			)
-		).toBeVisible();
-		await expect(
-			page.locator(
-				'role=region[name="Save panel"] >> role=checkbox[name="Test Page"]'
-			)
-		).toBeVisible();
-	} );
+	// Header template area should be hidden.
+	await expect(
+		editor.canvas.getByRole( 'document', {
+			name: 'Block: header',
+		} )
+	).toBeHidden();
 
-	test( 'toggle template preview', async ( { page, editor } ) => {
-		await draftNewPage( page );
-		await editor.openDocumentSettingsSidebar();
+	// Check order of paragraphs.
```

Comment:
This is now checking the order of the Content and Title blocks. There's already a comment below, so we might be able to remove this and the following line instead?